box.json

```diff
@@ -0,0 +1,42 @@
+{
+  "finder": [
+    {
+      "in": "src"
+    },
+    {
+      "name": "autoload.php",
+      "in": "vendor"
+    },
+    {
+      "name": [
+        "autoload_real.php",
+        "ClassLoader.php",
+        "autoload_static.php"
+      ],
+      "in": [
+        "vendor/composer"
+      ]
+    },
+    {
+      "name": [
+        "*.php",
+        "LICENSE"
+      ],
+      "exclude": [
+        "Tests",
+        "Tester"
+      ],
+      "in": [
+        "vendor/symfony/console",
+        "vendor/symfony/filesystem",
+        "vendor/symfony/process",
+        "vendor/symfony/stopwatch",
+        "vendor/symfony/polyfill-mbstring"
```

Comment:
Would it be possible to summarize this list in a `vendor/symfony`?  So we would juste have to update this file when adding packages from a new vendor